scripts/utils/generate_vif/constants.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+XML_ENCODING = "utf-8"
+XML_ELEMENT_NAME_PREFIX = "vif"
+XML_ROOT_ELEMENT_NAME = "VIF"
+XML_NAMESPACE_ATTRIBUTES = {
+    "xmlns:opt": "http://usb.org/VendorInfoFileOptionalContent.xsd",
+    "xmlns:xsi": "http://www.w3.org/2001/XMLSchema-instance",
+    "xmlns:vif": "http://usb.org/VendorInfoFile.xsd",
```

Comment:
Dead link already in a PR ?
![image](https://user-images.githubusercontent.com/9802583/198971902-e3b1dbcb-1d6f-4471-b168-fe93fa482b5d.png)

Reply:
I updated the links with working ones